packages/validate/src/ValidateMixin.js

```diff
@@ -48,6 +48,8 @@ export const ValidateMixin = dedupeMixin(
                   return import('../translations/en-AU.js');
                 case 'en-GB':
                   return import('../translations/en-GB.js');
+                case 'en-PH':
```

Comment:
you need to add a similar case to each component that has such a switch statement for the compatibility with `polymer-cli`, to find all the places in our code you can use this: https://github.com/ing-bank/lion/search?q=%22case+%27en%27%22&unscoped_q=%22case+%27en%27%22

you don't need to create a file everythere btw unless you know it has slightly different translations, so often you can just fallback to `en.js`, e.g.

```
case 'en-PH':
case 'en':	
  return import('../translations/en.js');
```

Reply:
Done updating the files